libs/components/progress-indicator/src/lib/modules/progress-indicator/progress-indicator-item/progress-indicator-item.component.html

```diff
@@ -12,21 +12,35 @@
   <div class="sky-progress-indicator-item-content">
     <div
       *ngIf="showTitle"
-      aria-level="2"
-      class="sky-progress-indicator-item-heading"
-      role="heading"
-      [ngClass]="{
-        'sky-deemphasized': statusName === 'pending'
-      }"
-      [skyThemeClass]="{
-        'sky-font-display-3': 'modern'
-      }"
+      class="sky-progress-indicator-item-heading-wrapper"
       skyTrim
     >
-      {{ formattedTitle }}
-      <span class="sky-control-help-container" skyTrim
-        ><ng-content select=".sky-control-help"></ng-content
-      ></span>
+      <span
+        aria-level="2"
+        class="sky-progress-indicator-item-heading"
+        role="heading"
+        [ngClass]="{
+          'sky-deemphasized': statusName === 'pending',
+          'sky-margin-inline-xs': !!helpPopoverContent
+        }"
+        [skyThemeClass]="{
+          'sky-font-display-3': 'modern'
+        }"
+      >
+        {{ formattedTitle }}
+      </span>
+      @if (helpKey || helpPopoverContent) {
+        <sky-help-inline
+          [helpKey]="helpKey"
+          [labelText]="title"
+          [popoverTitle]="helpPopoverTitle"
+          [popoverContent]="helpPopoverContent"
+        />
+      } @else {
+        <span class="sky-control-help-container" skyTrim
```

Comment:
Consumer unit tests may be expecting this element to be a child of `.sky-progress-indicator-item-heading`. I'd put this back where it was, but wrap it in reverse logic:
```
@if (!helpKey && !helpPopoverContent) {
```

Reply:
moved